bin/workflows/pycbc_make_coinc_search_workflow

```diff
@@ -155,7 +155,23 @@ precalc_psd_files = wf.setup_psd_workflow(workflow, analyzable_segs,
 bank_files = wf.setup_tmpltbank_workflow(workflow, analyzable_segs,
                                          datafind_files, output_dir="bank",
                                          psd_files=precalc_psd_files)
-hdfbank = wf.convert_bank_to_hdf(workflow, bank_files, "bank")
+
+# Determine the extension of the bank file taken as input
+cp = workflow.cp
+tmpltbankMethod = cp.get_opt_tags("workflow-tmpltbank", "tmpltbank-method", '')
+if tmpltbankMethod == "PREGENERATED_BANK":
+    pre_gen_bank = cp.get_opt_tags('workflow-tmpltbank',
+                                           'tmpltbank-pregenerated-bank', '')
+    bank_filename, ext = (pre_gen_bank.split('/')[-1].split('.'))
+
+# If the bank is a hdf or h5 file do not change the format. If not,
+# convert it to hdf
+if ext == 'hdf' or ext == 'h5' :
+    hdfbank = bank_files
+else :
+    hdfbank = wf.convert_bank_to_hdf(workflow, bank_files, "bank")
+logging.info("Input bank is a %s file", ext)
```

Comment:
@spxiwh If I understand it correctly, do you want it to do the part of checking what the extension of the bank is in `setup_tmpltbank_workflow` , and then if the bank is in hdf format, return as it is, if it is in xml call `convert_bank_to_hdf ` to convert it to hdf in `setup_tmpltbank_workflow` , and if any other format, just return an error.. ie. do all of the changes in `setup_tmpltbank_workflow` ?